.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,129 @@
+name: Bug report
+description: File a bug report
+title: '[BUG] '
+labels: ['bug', 'triage']
```

Comment:
We can also add [`type: Bug`](https://github.com/interledger/publisher-tools/issues?q=is%3Aissue%20state%3Aopen%20type%3ABug) here.